### DISCUSSION TITLE
"Open with" windows - small issue

### DISCUSSION THREAD

**Discussion by cjsrussell** [posit-dev/positron#4135] [Q&A]
Seems like a small issue from an older pre-production version or something, but when using "open with" on a script file on Windows it offers "Electron" instead of Positron.

Using version 2024.07.0

![image](https://github.com/user-attachments/assets/323937e2-4996-4d6a-933a-5c94859331a6)

**Comment by juliasilge** [comment]
Thanks for raising this! 🙌  I will add this specific example to #3671, which I believe is related.
